### PR TITLE
Update day-8.js

### PR DESCRIPTION
**Pull request by emcsquared2**

### CODE REVIEW

File: resources/js/day-8.js [hunk 1]
```diff
@@ -96,4 +96,4 @@ function executeFunction(){
 }
 }
 startQuizButton.onclick = startQuiz;
-submitButton.onclick = executeFunction;
+submitButton.onclick = executeFunction;
```
Comment (covolution) [inline resources/js/day-8.js:102]: Are you sure you want to change that?

**Review by covolution — APPROVED** [review]
Changes look good

File: resources/js/day-8.js (hunk 1 above)
Comment (emcsquared2) [reply resources/js/day-8.js:102]: Aha thanks for commenting!  I can see how it works now

**Review by emcsquared2 — COMMENTED** [review]
Just playing to understand how this works.

Wondering if you covolution can see this still and comment on the latest update.

**Pull request merged**